unit-test/addTask.test.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import addTask from './addTask.js';
```

Comment:
- [ ] Kindly use the original function. You can use mock for functions that persists `localstorage`